ui/js/constants.js

```diff
@@ -388,3 +388,10 @@ export const phAlertStatusMap = {
 export const phCompareBaseLineDefaultTimeRange = 86400 * 2;
 
 export const thBugSuggestionLimit = 20;
+
+export const thDefaultDetailsPct = 40;
+
+export const thNavBarSpacer = 62;
+
+export const thPinboardHeight = 100;
```

Comment:
I think similar to @ghickman's thoughts for Django's `settings.py`, the "only used in one file" type constants are probably best kept in the file in which they are used, since it makes it easier to follow the breadcrumbs, and avoids having a several hundred line long settings/constants file. What do you think? :-)

Reply:
Sure!  Yeah, that's a good call.  doing that now...